[Docs] Refresh "Installation" section of the book (#10221)
Sylius/Sylius: docs/book/installation/installation.rst

Initiating A New Sylius Project
-------------------------------

> Create your new project by running:

Reviewer: Why? :)

Reviewer: ```suggestion
To begin creating your new project using Sylius Standard Edition, run this command:
```

Author: Let's say its one of the first info I read about Sylius. How can I know what is `Sylius Standard Edition` and whether I want to use it or whatever the alternative is?

Author: ```suggestion
To begin creating your new project, run this command:
```

    Make sure to use PHP ^7.2. Using an older PHP version will result in installing an older version of Sylius.

> This will create a new Symfony project in the ``acme`` directory. Move to the project directory next:

Reviewer: ```suggestion
This will create a new Symfony project in the ``acme`` directory. Next, move to the project directory:
```

.. code-block:: bash

    $ cd acme

Sylius uses enviornment variables to configurethe connection with database and mailer services.
You can look up the default values in ``.env`` file and customise them by creating ``.env.local`` with variables you want to override.
For example, if you want to change your database name from the default ``sylius_%kernel.environment`` to ``my_custom_sylius_database``,
the contents of that new file should look like the following snippet:

.. code-block:: text

    DATABASE_URL=mysql://username:password@host/my_custom_sylius_database

After everything is in place, run the following command to install Sylius:

.. code-block:: bash

>     $ bin/console sylius:install

Reviewer: ```suggestion
    $ php bin/console sylius:install
```

In order to see a fully functional frontend you will need to install its assets.

> **Sylius** uses `Gulp`_ to build fronted assets using `Yarn`_ as a JavaScript package manager.

Reviewer: ```suggestion
**Sylius** uses `Gulp`_ to build frontend assets using `Yarn`_ as a JavaScript package manager.
```

Having Yarn installed, go to your project directory to install the dependencies:

.. code-block:: bash

    $ yarn install

> Then build the frontend assets by running:

Reviewer: goodbye `simple` 👋 😢

Accessing the Shop
------------------

We strongly recommend using the Symfony built-in web server by running the ``php bin/console server:start``
command and then accessing ``http://127.0.0.1:8000`` in your web browser to see the shop.

.. note::

    The localhost's 8000 port may be already occupied by some other process.
    If that happens, please try using a different port - ``php bin/console server:start 127.0.0.1:8081``.

>     Want to know more about using a built-in server, see `here <http://symfony.com/doc/current/cookbook/web_server/built_in.html>`_.

Reviewer: ```suggestion
    Get to know more about using a built-in server `here <http://symfony.com/doc/current/cookbook/web_server/built_in.html>`_.
```

Author: 👍, can also improve the accessibility by creating a more meaningful link than "here".
